### PR TITLE
fix: fixing FPs

### PR DESCRIPTION
**Pull request by azurit**
Parameter `goto` indicates where the user should be redirected after doing some action. Example of value doing FP:

`goto=index.php?route=/sql&db=INFORMATION_SCHEMA&table=VIEWS&sql_query=select+VIEW_DEFINITION+%0D%0Afrom+INFORMATION_SCHEMA.VIEWS%0D%0Awhere+TABLE_NAME+%3D+%27wp_wpdatatable_14%27%3B&server=3`

### CODE REVIEW

**Review by EsadCetiner — APPROVED** [review]
LGTM

**Comment by EsadCetiner** [comment]
Looks like I don't have write access so I can't merge this PR

**Comment by azurit** [comment]
Thanks, merging.

**Comment by azurit** [comment]
@dune73 Can you give write access to this repo to @EsadCetiner? Thanks.

**Pull request merged**
